CopilotKit/packages/runtime/src/service-adapters/experimental/empty/empty-adapter.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * CopilotKit Empty Adapter
+ *
+ * This adapter does nothing
+ * Ideal if you want to be sure your front-end doesn't connect to any 3rd party LLM,
+ * except your GraphLang chain.
+ */
```

Comment:
```
 * ## Example
 *
 * ```ts
 * import { CopilotRuntime, ExperimentalEmptyAdapter } from "@copilotkit/runtime";
 *
 * const copilotKit = new CopilotRuntime();
 *
 * return new ExperimentalEmptyAdapter();
 * ```
 */
```

Reply:
I committed the GitHub committable part and then added this example. Check it out